oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -12,11 +16,66 @@ def __init__(self, project_item):
         self._item = project_item
 
     def populate_fields(self):
+        # Elements used in MODS directly on the ProjectItem
         self._populate_title()
         self._populate_identifier()
+        self._populate_relation()
+
+        self._populate_language()
+        self._populate_format()
+        self._populate_description()
 
     def _populate_title(self):
         self.title = self._item.title
 
     def _populate_identifier(self):
         self.identifiers.extend([mods.Identifier(text=self._item.ark)])
+
+    def _populate_relation(self):
+        if self._item.relation:
+            self.related_items.extend([mods.RelatedItem(title=self._item.relation)])
+
+    def _populate_language(self):
+        for ilu in ItemLanguageUsage.objects.filter(item=self._item):
+            lang = mods.Language()
+            lang.terms.append(mods.LanguageTerm(text=ilu.value))
+            self.languages.append(lang)
+
+    def _populate_format(self):
+        format = Format.objects.filter(item=self._item).first()
+        if format:
+            self.create_physical_description()
+            self.physical_description.extent = format.value
+
+    def _populate_description(self):
+        descriptions = Description.objects.filter(item=self._item)
+
+        # Similar note element behaving qualifiers
+        type_labels = {
+            "biographicalnote": "Biographical Information",
+            "interviewerhistory": "Interviewer Background and Preparation",
+            "personpresent": "Persons Present",
+            "place": "Place Conducted",
+            "processinterview": "Processing of Interview",
+            "supportingdocuments": "Supporting Documents",
+        }
+
+        for desc in descriptions:
+            desc_type = desc.type.type.lower()
+
+            if desc_type == "abstract":
+                self.create_abstract()
+                self.abstract.text = desc.value
+
+            elif desc_type in type_labels.keys():
+                type_value = (
+                    "biographical" if desc_type == "biographicalnote" else desc_type
+                )
+                self.notes.append(
+                    mods.Note(
+                        type=type_value, label=type_labels[desc_type], text=desc.value
+                    )
+                )
+
+            else:
```

Comment:
@kjallen I thought we agreed to exclude admin notes?  This still allows them through, via the `else`, as generic `mods.Note` elements.  Using http://127.0.0.1:8000/item/9603 , which has 2 admin notes and one ToC:
```
<?xml version="1.0" encoding="UTF-8"?>
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3">
  <mods:titleInfo>
    <mods:title>SESSION 1 (3/13/2017)</mods:title>
  </mods:titleInfo>
  <mods:identifier>21198/zz002kc9zw</mods:identifier>
  <mods:note>Persons Present: Harris Hayes and Collings</mods:note>
  <mods:note>Place: Ann Harris Hayes's home in Los Angeles</mods:note>
  <mods:note>Early life and education--Becomes a mother--Works as an interior designer--Sells her own art work--Has an interest in studying to become an architect, but is unable to pursue this direction--Begins work at Universal Studios--Benefits from pressure on studios to hire more women--The reporting structure and work routine in the are of production design at the time--Begins to freelance when the Universal art department is dissolved--Stephen J. Cannell Productions--The job duties of a set designer--Begins work as an art director.</mods:note>
</mods:mods>
```

Reply:
Yes, confirming admin should not show up. Reviewing.

Reply:
Added `exclude` filter for `adminnote` and `tableOfContents`
`else` catch all uses generic `mods:note` for `note` type and if new types added in future.